llm/deepseek-r1/README.md

```diff
@@ -62,14 +61,30 @@ run: |
 
 ```
 
+Adjust the `accelerators` and `num_nodes` to fit your needs. Common configurations include:
+
+| **GPU**          | **Num Nodes** |
+|------------------|---------------|
+| H200:8           | 1             |
+| H100:8           | 2             |
+| A100-80GB:8      | 4             |
+| A100:8           | 8             |
+
+.. note::
```

Comment:
This won’t work with readme? Should we make this a short footnote or just use a quote box?

Also, this should be moved after the place we talk about the start time.